README.md

```diff
@@ -56,7 +57,7 @@ Here you can find guides and examples for common scenarios:
 
 ## Contributing
 
-Read below to learn how to take part in improving Theia:
+Read below to learn how to take part in improving Eclipse Theia:
```

Comment:
For the sake of simplicity, I'd prefer to leave this as is.

Reply:
done